ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -1029,10 +1029,12 @@ private static ConfigDef buildConfigDef(final ConfigGeneration generation) {
         ).define(
             KSQL_UDF_SECURITY_MANAGER_ENABLED,
             ConfigDef.Type.BOOLEAN,
-            true,
+            // SecurityManager is deprecated from java 21 onwards.
+            false,
```

Comment:
Should be removing this? cc-spec-ksql and cp customers - would they be having configuration written as `KSQL_UDF_SECURITY_MANAGER_ENABLED=true` in ksql-server.properties.